conda-dist/recipe/meta.yaml

```diff
@@ -14,43 +14,45 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
     - xspec-modelsonly
-    - cfitsio
+    - cfitsio==3.410
     - ccfits
-    - wcslib
-
+    - wcslib==5.18
+    - krb5==1.14.6
 
   host:
     - python
     - setuptools
     - pip
     - numpy
     - pyyaml==3.13
-    - astropy
-    - scipy
+    #- astropy
+    #- scipy
     - numdifftools
-    - hdf5
-    - pytables
+    #- hdf5
+    #- pytables
     - pandas
-    - cfitsio
+    - cfitsio==3.410
     - ccfits
-    - wcslib
+    - wcslib==5.18
     - xspec-modelsonly
+    - krb5==1.14.6
 
   run:
-    #- python
+    - python
   #  - libgfortran
     - numpy
     - pyyaml==3.13
     - astropy
-    - scipy
+    #- scipy
     - numdifftools
-    - hdf5
+    #- hdf5
     - pytables
     - pandas
     - dill
-    - cfitsio
+    - cfitsio==3.410
     - ccfits
     - wcslib
+    - krb5==1.14.6
 
 build:
   script: python -m pip install --verbose --no-deps --ignore-installed .
```

Comment:
version pinning is usually considered a bad idea because it makes the code more fragile... but I think in our case we need it.

Reply:
Yes, I strongly agree with you. Unfortunately up to now that's the only way I found to solve the conflicts. As soon as we find a different way we can certainly relax this constrain.